src/order/bounded_lattice.lean

```diff
@@ -309,6 +309,9 @@ noncomputable instance Prop.linear_order : linear_order Prop :=
 @[simp]
 lemma le_iff_imp {p q : Prop} : p ≤ q ↔ (p → q) := iff.rfl
 
+@[simp] lemma sup_Prop_eq (p q : Prop) : p ⊔ q = (p ∨ q) := rfl
+@[simp] lemma inf_Prop_eq (p q : Prop) : p ⊓ q = (p ∧ q) := rfl
```

Comment:
Should these be `=` or `↔` to match the `le_iff_imp` above?

Reply:
Probably we should have `(⊔) = or` instead.